File: Exercicio Diagrama de Classes/Classes_JS/PessoaFisica.js
import { Pessoa } from "./Pessoa.js";

export class PessoaFisica extends Pessoa {
  constructor(identificador, endereco, nome) {
    super(identificador, endereco);
    this.nome = nome;
  }

  exibirDados() {
    return `${super.exibirDados()}\nNome (Pessoa Física): ${this.nome}`;
  }
}

File: Exercicio Diagrama de Classes/Classes_Python/pessoa_fisica.py
from pessoa import Pessoa

class PessoaFisica(Pessoa):
    def __init__(self, identificador: str, endereco: str, nome: str):
        super().__init__(identificador, endereco)
        self.nome = nome

    def exibir_dados(self):
        super().exibir_dados()
        print(f"Nome (Pessoa Física): {self.nome}")